src/InstallationValidator.Core/Domain/ComparisonSettings.cs

```diff
@@ -1,10 +1,24 @@
-﻿using System.Collections.Generic;
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text.RegularExpressions;
 using InstallationValidator.Core.Assets;
+using OSPSuite.Core.Domain;
+using OSPSuite.Core.Extensions;
+using OSPSuite.Utility.Extensions;
+using static OSPSuite.Core.Domain.Constants;
 
 namespace InstallationValidator.Core.Domain
 {
    public class ComparisonSettings
    {
+      private HashSet<string> _exclusionHash = new HashSet<string>();
+      private IReadOnlyList<Regex> _regexExclusions = new Regex[] { };
+      private bool _hasExclusion;
+      private static readonly string ALL_BUT_PATH_DELIMITER = $"[^{ObjectPath.PATH_DELIMITER}]*";
```

Comment:
@Yuri05 This is the same code as in Core. Did not wnt to try to refactor it in this case. IF we have to use sthg like this a 3rdtime I will